torch/utils/data/datapipes/iter/tobytes.py

```diff
@@ -0,0 +1,21 @@
+from typing import Tuple
+from torch.utils.data import IterDataPipe
+
+
+class ToBytesIterDataPipe(IterDataPipe[Tuple[str, bytes]]):
```

Comment:
Why we need this DataPipe to read bytes with a maximum limit?

Reply:
IMO, it makes more sense to have a Datapipe to read specific bytes in each iteration. Then close the stream when deplete it.

Reply:
fyi, agree -- I change the parameter to be 'chunk' which is the size limit (def=None) on each read iteration. Per Vitaly, no close stream, rely on ref counter here.